### ISSUE TITLE
As a publisher I will decide for myself whether an image should have alt text or not depending on if it's presentational or not 

### ISSUE THREAD

**Issue by wenche** [equinor/energyvision#369]
How could we achieve and avoid a lot on empty alt tags that should have had an alt text. How much noise will this generate with tools like Axe. How can we explain this in an optimal way inside the Studio. Should we at least use a warning?

## Acceptance criteria
1. [Criteria 1]
1. [Criteria 2]

## In order to ensure good quality we should (Keep the ones we need for this US)
- [ ] Perform integration testing
- [ ] Perform functional testing
- [ ] Extra focus on usability
- [ ] Test for portability
- [ ] Test for accessibility
- [ ] Create automated acceptance test/smoke test
- [ ] Perform performance and load test
- [ ] Perform user testing
- [ ] Perform security testing

**Comment by SvSven** [comment]
@nicholas-haggerty @meols 

Added a switch to the image component where you can set an image as being decorative - disabling the requirement for an alt tag. If the image is set as decorative it will render an empty alt tag like `alt=""` in the web in compliance with the accessibility guidelines.

![decorative_img_switch.gif](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/2a659ea8-2df4-419c-9eb8-6faf180c7f89)

**Comment by nicholas-haggerty** [comment]
@SvSven Looks great. Liking the Gif too!
